x-pack/plugins/uptime/common/rules/uptime_rule_field_map.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const uptimeRuleFieldMap = {
+  // common fields
+  'monitor.id': {
+    type: 'keyword',
+  },
+  'url.full': {
+    type: 'keyword',
+  },
+  reason: {
+    type: 'text',
+  },
+  // monitor status alert fields
+  'error.message': {
+    type: 'text',
+  },
+  'agent.name': {
+    type: 'keyword',
+  },
+  'monitor.name': {
+    type: 'keyword',
+  },
+  'monitor.type': {
+    type: 'keyword',
+  },
+  'observer.geo.name': {
+    type: 'keyword',
+  },
+  // tls alert fields
+  'cert.count': {
```

Comment:
@shahzad31 We can't use the ecs fields here, since the fields represent a single instance of a cert, but these are meant to represent all the found expiring/aging certs to the entire alert. (Also this is supposed to be `cert_status.count`, like the rest of the items below).

Reply:
++ that makes senese